### PR TITLE
Clearer warnings 4 gene search - 2 -

### PR DESCRIPTION
**Pull request by northwestwitch**
Some changes I didn't manage to add to a previous PR since I've merged it by mistake!

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [ ] tests executed by

### CODE REVIEW

**Comment by northwestwitch** [comment]
@mhkc do you have time for this easy review?

**Review by dnil — CHANGES_REQUESTED** [review]
How about these edits? Feel free to re-edit. And the request changes is only because you asked for it. I find it usually works out perfectly fine to Approve-with-some-caveats, but I can try this as well. 😉

File: scout/server/blueprints/variants/controllers.py [hunk 1]
```diff
@@ -1007,7 +1007,7 @@ def check_form_gene_symbols(
 
     errors = {
         "non_clinical_symbols": {
-            "info": "Genes not included in case gene panels",
+            "info": "Genes not included in gene panels versions used when loading variants of this case",
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/variants/controllers.py:1010]: They're good. I'll apply!

File: scout/server/blueprints/variants/controllers.py [hunk 2]
```diff
@@ -1019,7 +1019,7 @@ def check_form_gene_symbols(
             "gene_list": not_found_ids,
         },
         "outdated_symbols": {
-            "alert": "Case gene panels contain an outdated symbol for genes",
+            "info": "Gene panels versions used for loading variants of this case contain outdated genes symbols",
```
[the remark this answers is not in the document]
Comment (northwestwitch) [reply scout/server/blueprints/variants/controllers.py:1022]: I think users didn't understand what clinical list is so I'd avoid the term

**Comment by northwestwitch** [comment]
> I find it usually works out perfectly fine to Approve-with-some-caveats, but I can try this as well.

To be perfectly honest with you the merging of the previous PR went like this: I was working on the sofa and while leaning in to reach a bag of chips, I inadvertently push the merge button with a part of my body that is not a finger ..

**Review by dnil — APPROVED** [review]

**Pull request merged**